### DISCUSSION TITLE
JSON LD “graph” store

### DISCUSSION THREAD

**Discussion by entrepeneur4lyf** [cline/cline#369] [Feature Requests]
Looking through the code, I didn’t see any kind of vector store. Did I miss that? Anyway… you might consider using JSON-LD to assist the model to remember context without having all the context in memory. 

* https://www.w3.org/TR/json-ld11/
* https://jsoncrack.com/
* https://paulfrazee.medium.com/pauls-notes-on-how-json-ld-works-965732ea559d

**Comment by saoudrizwan** [comment]
Thank you for the links! (And no vectorizing happening here, claude starts off each task seeing your project's file structure and it turns out this alone gives him quite a lot of insight)
